### ISSUE TITLE
jmk files not supported

### ISSUE THREAD

**Issue by Sophrinix** [Glavin001/atom-beautify#1355]
jmk files are jake make files. They are javascript. Is there a simple way to configure this library to let it know to treat jmk files as js files?

**Comment by prettydiff** [comment]
Yes, submit a pull request modifying this file: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/javascript.coffee
